src/net/dmytro/javapro/homework5/task2/Team.java

```diff
@@ -0,0 +1,48 @@
+package net.dmytro.javapro.homework5.task2;
+
+import net.dmytro.javapro.homework5.task2.obstacle.Obstacle;
+import net.dmytro.javapro.homework5.task2.participant.liveparticipants.Cat;
+import net.dmytro.javapro.homework5.task2.participant.liveparticipants.Human;
+import net.dmytro.javapro.homework5.task2.participant.Participant;
+import net.dmytro.javapro.homework5.task2.participant.Robot;
+
+public class Team {
+    private String name;
+    private Participant participants[];
+
+    public Team(String name) {
+        this.name = name;
+    }
+
+    public Team(String name, Participant ... participantGiven) {
+        this.name = name;
+        this.participants = participantGiven;
+    }
+    public void getTeamInfo(){
+        System.out.println("Team name " + this.name);
+        for (Participant participant : participants){
+            if (participant instanceof Cat){
+                System.out.println("Cat " + participant.getName());
```

Comment:
it is good idea to create such class, just need to remove those ifs ...
better to override getName() method in classes to return  for example `"Cat + " + name`, same for another classes ... 

in result you will have much better loop. 
```
for (Participant participant : participants){ 
  System.out.println("Cat " + participant.getName());
}
```
the reason why better to avoid such constructions is ... in case you want to add new participant, you will create class implement methods, and there is chance you can forget to to add new if here, in this loop.